schema/dcschema_enum_instances.mcf

```diff
@@ -123,11 +123,1601 @@ description: "Category of race or ethnicity group used by Opportunity Insights t
 typeOf: dcs:OI_RaceCodeEnum
 
 ###---- begin BEA enum instances ----###
-Node: dcid:ConsumerGoodsAndServices
```

Comment:
Did you mean to delete this node?

Reply:
Yes because it is not used, or do we still want to keep it?